contribute/documentation/README.md

```diff
@@ -1,7 +1,14 @@
 # Contribute to our documentation
 
-Welcome. We're glad you're here to help make our technical documentation even better.
+Welcome! We're glad you're here.
```

Comment:
https://developers.google.com/style/periods#exclamation-points

Reply:
I'm aware of this guideline but I'm disregarding it in this case. A period after a one-word exclamation conveys a lack of enthusiasm, which is not appropriate. If you disagree let's take to style council in the future.

Reply:
Sure, it's fine to disregard the style rules if it's done with consideration. To help future contributors, we should either:
- document the reasons one might disregard this rule if it's a general exception
- add a comment explaining the decision in the source file if it's a one-off

Reply:
I've added the ! exception to the doc style roundtable sheet for discussion. Keeping the ! at this moment.